packages/admin-ui/src/filters/option.tsx

```diff
@@ -0,0 +1,36 @@
+import type { ReactNode } from 'react'
+import type { ListState } from '@react-stately/list'
+
+import React, { useRef } from 'react'
+import { useOption } from '@react-aria/listbox'
+import { tag } from '@vtex/admin-ui-react'
+import { focusVisible } from '@vtex/admin-ui-core'
+import type { FilterItem } from './filter.state'
+
+export const Option = ({
+  item,
+  state,
+  inputRenderer,
+}: {
+  item: { key: string | number; rendered: ReactNode }
+  state: ListState<FilterItem>
+  inputRenderer: ({ isSelected }: { isSelected: boolean }) => ReactNode
+}) => {
```

Comment:
Avoid param destructuring and also implement the component interface
```suggestion
export const Option = (props: OptionProps) => {
```